ChangeLog.md

```diff
@@ -151,6 +151,10 @@
 
   * Remove old temporary upgrade flag `tmp_upgraded_iq_status` (Matt West).
 
+  * Fixed a bug in tools/generate_uuid.py that added a uuid for files in the element subdirectory (Pavitra Shadvani).
+
+  * Added a verbose flag to tools/generate_uuid.py to show all the files changed.(Pavitra Shadvani).
+
```

Comment:
Start ChangeLog entries with exactly "Fix", "Add", etc, as already there.

Group all the "Fix" entries together, all the "Add" entries together, etc.

Put backticks around filenames (the ChangeLog is Markdown).

You have an extra period before the `(` character in the last entry above.

Capitalize UUID.